avail-subxt/build_api.sh

```diff
@@ -2,7 +2,7 @@
 echo "⛓ Installing SubXt..."
 cargo install --git https://github.com/paritytech/subxt --tag v0.29.0 subxt-cli || true 
 echo "🔨 Generating Avail-SubXt API from localhost..."
-subxt codegen \
+subxt codegen --version 14 \
```

Comment:
Our new node also supports version 15, but Subxt defaults to using v15. We're specifying v14 because v15 hasn't stabilized yet and would introduce breaking changes in LC compatibility